app/router.js

```diff
@@ -8,6 +8,9 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('about');
   this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:title' });
```

Comment:
Could we add a `slug` field that would have this pre-built as part of the Mirage data?  It tends to fit better with the way it would generally be done.  Either that or use ids to help expose those and make it clearer what is going on ...